src/components/Header/UserSettings.ts

```diff
@@ -0,0 +1,188 @@
+import { css, html, LitElement } from "lit";
+import {
+  customElement,
+  property,
+  query,
+  queryAll,
+  state,
+} from "lit/decorators.js";
+import { localized, msg } from "@lit/localize";
+import { theme } from "../../utils/theme.ts";
+
+@customElement("bkd-user-settings")
+@localized()
+export class UserSettings extends LitElement {
+  @property()
+  currentLocale = "de";
+
+  @state()
+  protected open = false;
+
+  @query("button")
+  private menuButton: any;
+  @queryAll("a")
+  private menuLinks: any;
+
+  static styles = [
+    theme,
+    css`
+      :host {
+        display: flex;
+        position: relative;
+      }
+
+      button {
+        background: transparent;
+        border: none;
+        cursor: pointer;
+      }
+
+      ul {
+        position: absolute;
+        right: 0;
+        border: 1px solid var(--bkd-func-bg-grey);
+        list-style-type: none;
+        padding: 0.625rem 0;
+        margin-top: calc(32px + 0.5rem);
+        background: var(--bkd-func-bg-white);
+        z-index: 1;
+        min-width: 12rem;
+      }
+
+      a {
+        font-size: 0.875rem;
+        letter-spacing: 0.01rem;
+        word-spacing: 0.025rem;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 2.5;
+        padding: 0 1.5rem;
+        display: flex;
+        align-items: center;
+        gap: 0.25rem;
+        height: 100%;
+        width: 100%;
+        text-decoration: none;
+        color: var(--bkd-func-fg-black);
+      }
+
+      a img {
+        margin-left: -5.25px;
+      }
+
+      a:hover {
+        color: var(--bkd-brand-red);
+        background: var(--bkd-brand-light-sand);
+        border-left: 6px solid var(--bkd-brand-red);
+        font-weight: 700;
+        padding: 0 calc(1.5rem - 6px);
+      }
+    `,
+  ];
+
+  profile() {
+    return html`<a href="#">${msg("Mein Profil")}</a>`;
+  }
+
+  settings() {
+    return html`<a href="#">${msg("Einstellungen")}</a>`;
+  }
+
+  videos() {
+    const playlist =
+      this.currentLocale === "de"
+        ? "PLLDtLiOuctbx-_EQWgWqTO1MRbX845OEf"
+        : "PLLDtLiOuctbyEegnquAkaW4u8cm62lFAU";
+
+    return html`<a
+      href=${`https://www.youtube.com/playlist?list=${playlist}`}
+      target="_blank"
+      ><img src=${"/icons/external-link.svg"} alt="" width="24" height="24" />
+      ${msg("Video-Tutorials")}</a
+    >`;
+  }
+
+  logout() {
+    return html`<a href="#"
+      ><img src=${"/icons/logout.svg"} alt="" width="24" height="24" />${msg(
+        "Logout"
+      )}</a
+    >`;
+  }
+
+  render() {
+    return html`
+      <button
+        type="button"
+        @click=${() => this.toggle()}
+        aria-label=${msg("Menü Benutzereinstellungen")}
```

Comment:
:+1: 

Vielleicht könntest du noch `aria-expanded` auf `this.open` setzen?

Und wenn wir schon dabei sind, würde ein `aria-haspopup="menu"` auf dem `<button>`, `role="menu"` auf dem `<ul>` und `role="menuitem"` auf dem `<li>` hier Sinn machen?